doc/release_instructions.md

```diff
@@ -84,30 +78,31 @@ If you're doing a custom release (off a branch that isn't master), [skip to the
     > git push origin master
     ```
 
-### If the release failed
-You'll need to do two things: remove the relevant tags in GitHub, and re-build the docs in ReadTheDocs
+8) Regenerate the golden deployment manifests: `./etc/testing/deploy-manifests/validate.sh --regenerate`
 
-1) Removing the tag in GitHub
+9) Commit the changes:
 
-  You'll need to delete the *release* and the *release tag* in github. Navigate to
-  `https://www.github.com/pachyderm/pachyderm` and click on the *Releases* tab.
-  Click on the big, blue version number corresponding to the release you want to
-  delete, and you should be redirected to a page with just that release, and red
-  "Delete" button on the top right. Click the delete button
+  ```
+  > git commit -a -m"Regenerate golden deployment manifests for $(pachctl version --client-only)"
+  > git push origin master
+  ```
 
```

Comment:
Its worth adding another setup to post the release notes to github and check github and docker hub accounts to check the release has been posted.

Reply:
the release process will fail if either of those happen, should I still add the note still?